interop-twitter/jvm/src/main/scala/scalaz/zio/interop/twitter.scala

```diff
@@ -33,4 +34,25 @@ package object twitter {
         }
       }
   }
+
+  implicit class RuntimeOps[R](private val runtime: Runtime[R]) extends AnyVal {
+    def unsafeRunToTwitterFuture[E <: Throwable, A](io: ZIO[R, E, A]): Future[A] =
```

Comment:
I would just hard code `R = Throwable` because it can automatically be any subtype of `Throwable` due to variance of `E`.

Reply:
@jdegoes you mean `E = Throwable` like `ZIO[R, Throwable, A]`?